spec/services/usda_service_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe UsdaService do 
+  it 'searches foods by keyword' do 
+    search = UsdaService.food_search('sweet potatoes')
+
+    expect(search).to be_a Hash 
+    expect(search).to have_key(:totalHits) 
+    expect(search).to have_key(:foods)
+    expect(search[:foods].count).to eq(10)
+  end
+end
```

Comment:
We also want to test that the keys are in the position where your code is expecting them to be. It's possible that the USDA response format could change in the future, but your code relies on `totalHits` and `foods` being in a particular place -- so write tests that make sure that the keys your code relies on are in a particular format.